clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ListOffsetsEvent.java

```diff
@@ -47,13 +46,10 @@ public ListOffsetsEvent(Map<TopicPartition, Long> timestampToSearch,
     /**
      * Build result representing that no offsets were found as part of the current event.
      *
-     * @return Map containing all the partitions the event was trying to get offsets for, and
-     * null {@link OffsetAndTimestamp} as value
+     * @return empty map
      */
     public <T> Map<TopicPartition, T> emptyResults() {
-        Map<TopicPartition, T> result = new HashMap<>();
-        timestampsToSearch.keySet().forEach(tp -> result.put(tp, null));
-        return result;
+        return Map.of();
```

Comment:
@xijiu could you please add comments for this behavior? it is used to align with classic consumer

Reply:
Sure, I have added it, PTAL